### ISSUE TITLE
Make Install scripts version agnostic

### ISSUE THREAD

**Issue by giobart** [oakestra/oakestra#375]
## Short
The current install scripts are coming from a folder in our repository: https://github.com/oakestra/oakestra/tree/develop/scripts
If I need to install a version of Oakestra from the main branch, but I changed my install script from develop, that should not create an incompatibility with the older version. Currently in the documentation we use a script from the develop branch. 

## Proposal 
Each branch hosts the install script that works for that version. The install script reference version MUST be main. If via version selector `OAKESTRA_VERSION` the developer selects a different version, then the script from that branch must be executed. 

### Root Orchestrator

Root install wrapper (install-root.sh) hosted in oakestra.github.io
New install command: `curl -sfL oakestra.io/install-root.sh | sh -` 

```
#!/bin/bash
#If version not set, use latest version
if [ -z "$OAKESTRA_VERSION" ]; then
    OAKESTRA_VERSION=$(curl -s https://raw.githubusercontent.com/oakestra/oakestra/main/version.txt)
fi
curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_VERSION/scripts/StartOakestraRoot.sh | OAKESTRA_BRANCH=$OAKESTRA_VERSION sh -
```

### Cluster Orchestrator

Cluster install wrapper (install-cluster.sh) hosted in oakestra.github.io
New install command: `curl -sfL oakestra.io/install-cluster.sh | sh -` 

```
#!/bin/bash
#If version not set, use latest version
if [ -z "$OAKESTRA_VERSION" ]; then
    OAKESTRA_VERSION=$(curl -s https://raw.githubusercontent.com/oakestra/oakestra/main/version.txt)
fi
curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_VERSION/scripts/StartOakestraCluster.sh | OAKESTRA_BRANCH=$OAKESTRA_VERSION sh -
```

### Worker Node

Worker install wrapper (install-worker.sh) hosted in oakestra.github.io
Install command: `curl -sfL oakestra.io/install-worker.sh | sh -` 

```
#!/bin/bash
#If version not set, use latest version
if [ -z "$OAKESTRA_VERSION" ]; then
    OAKESTRA_VERSION=$(curl -s https://raw.githubusercontent.com/oakestra/oakestra/main/version.txt)
fi
curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_VERSION/scripts/InstallOakestraWorker.sh | sh -
```

### Standalone Getstarted

Worker install wrapper (getstarted.sh), **already** hosted in oakestra.github.io
Install command: `curl -sfL oakestra.io/getstarted.sh | sh -` 

```
#!/bin/bash
#If version not set, use latest version
if [ -z "$OAKESTRA_VERSION" ]; then
    OAKESTRA_VERSION=$(curl -s https://raw.githubusercontent.com/oakestra/oakestra/main/version.txt)
fi
curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_VERSION/scripts/InstallOakestraFull.sh | OAKESTRA_BRANCH=$OAKESTRA_VERSION sh -
```

### Notes

- This solution uniforms the branch selection mismatch (before OAKESTRA_VERSION and OAKESTRA_BRANCH) and now only the keyword OAKESTRA_VERSION is used

- These wrappers make the install command nicer, and they only download the main install script from the branch. 

- These wrappers do not require a minor version or an updated of the pre-existing scripts. We only change the documentation.

- Each branch will manage the installation the way they prefer

## Status
Describe the current status of this proposal. E.g., looking for feedback, searching for a solution, development, testing. Try to be concise but descriptive

## Checklist
- [ ] Discussed
- [ ] Solved
- [ ] Tested

**Comment by smnzlnsk** [comment]
Sounds good!

**Comment by nitindermohan** [comment]
This definitely looks like a good workaround

**Comment by giobart** [comment]
Short demo of how it works:

![ScreenRecording2024-12-20at10 39 34-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/ea50c164-e106-4cee-b061-aaa78f18789c)

Current compatibility:

Worker Node:                v0.4.301 ✅ alpha-v0.4.302 ✅ alpha-v0.4.400 ✅ 
Cluster Orchestrator:    v0.4.301 🚫 alpha-v0.4.302 ✅ alpha-v0.4.400 ✅
Root Orchestrator:        v0.4.301 🚫 alpha-v0.4.302 ✅ alpha-v0.4.400 ✅

My reccomendation: we can merge this change as part of the next release, but we should update the old v0.4.301 wiki to use the legacy root startup scripts. Back when we released v0.4.301 the root standalone startup was still mostly done by docker-compose manually. The fancy script that must be referenced from a different branch. 
A new PR to the docs is about to arrive.

**Comment by giobart** [comment]
Merged into dev docs
